api/graphql/schema.graphql

```diff
@@ -1759,6 +1761,11 @@ type Mutation {
     @update
     @guard
     @can(ability: "update", find: "id")
+  deleteSkill(id: UUID! @whereKey): Skill
+    @delete
+    @validator(class: "App\\GraphQL\\Validators\\DeleteSkillValidator")
+    @can(ability: "delete", find: "id")
```

Comment:
The SkillPolicy.delete method doesn't take a skill argument, so you can leave out `find` here.